src/ActionModal/components/UndelegationModal.vue

```diff
@@ -4,9 +4,11 @@
     ref="actionModal"
     :validate="validateForm"
     :amount="0"
-    title="Unstake"
+    :title="isRedelegation ? 'Restake' : 'Unstake'"
```

Comment:
apologies for the all the miscommunication here.

i think we should remove redelegation from this component as well. both delegation and undelegation modals should include to and from fields that are dropdowns.